src/main/kotlin/vm/VM.kt

```diff
@@ -131,6 +159,11 @@ class VM(
                     val args = (pop() as VList).v.map { (it as VString).v }
                     push(exe.getLambda(block, c.vThis, args, withVars))
                 }
+                O_ERRVAL -> {
```

Comment:
We have to construct VErr at runtime (instead of just O_VAL'ing a VErr like most types) because the message could be a runtime expression rather than a hardcoded string.